include/storage/shared_datatype.hpp

```diff
@@ -49,6 +83,10 @@ struct SharedDataLayout
         BEARING_BLOCKS,
         BEARING_VALUES,
         ENTRY_CLASS,
+        LANE_DATA_ID,
+        TURN_LANE_DATA,
+        LANE_DESCRIPTION_OFFSETS,
+        LANE_DESCRIPTION_MASKS,
```

Comment:
Add an static_assert here that check that `NUM_BLOCKS` and the size of `block_id_to_name` are the same. otherwise we might have errors in the error reporting.